src/app/shared/users-list/users-list.component.html

```diff
@@ -0,0 +1,7 @@
+<div *ngFor="let user of usersList">
+    <app-user [user]="user" [route]="route" *ngIf="
+    (route === 'active-users' && !user.isDeleted)||
+    (route === 'deleted-users' && user.isDeleted)||
+    (route == 'manage-users')">
+    </app-user>
+</div>
```

Comment:
This html should be like 

```
<div *ngFor="let user of usersList">
    <app-user [user]="user"></app-user>
</div> 
```

Any logic of on how the data and what kind of data should be passed should be done on the parent component itself.

Reply:
did this